.gitignore

```diff
@@ -19,6 +19,7 @@ vendor
 !/mod/dashboard/
 !/mod/developers/
 !/mod/diagnostics/
+!/mod/discussion/
```

Comment:
I changed it to singular today because pretty much everything was singular when this was part of the groups plugin.